src/main/python/api/house_service.py

```diff
@@ -0,0 +1,18 @@
+import flask
+from flask import jsonify
+import pymysql
+import sys
+from pathlib import Path
+
+sys.path.insert(1, str(Path(__file__).parent.parent))
```

Comment:
I keep seeing this in several files, and it looks like a consequence of how you've structured the directory tree -- which is very Java-esque. While there's technically not anything *wrong* with that, it's not considered idiomatic, and makes you need to use workarounds like this more often than not.

I can see you know how to use `__init__.py` files to help enforce `PATH` precedence anyway, so leveraging that as close to top-level as possible would make this cleaner.

Reply:
I'll module-ify these.